modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -10,15 +10,16 @@ This guide is for cloud deployments using the cloud-based version of Runtime Man
 
 CloudHub provides a variety of tools to architect your integrations and APIs so they are maintainable, secure, and scalable. This guide covers the basic network architecture, DNS, and firewall rules.
 
-image::cloudhub-networking-guide-1.jpg[CloudHub+Networking+Guide-1]
+.The image shows the CloudHub Network Overview.
+image::ch-networking-guide.png[CloudHub Network Overview]
```

Comment:
Questions about the diagram:

- Why do the lines that flow either from a node to text or text to node (I'm not sure which) have the same appearance as the lines between nodes?
- Why is there a stronger shadow on the load balancer than on the worker nodes? Why are you using shadows? They don't seem necessary.
- What is the relationship of the text next to the load-balancer node to that node?
- Why do the two lines extending from the worker nodes converge on empty space?
- If the node labeled "Optional VPC" is optional, where are the workers if that VPC is not used?
- What is the relationship between the load-balancer node and the worker nodes?
- In which direction is data flowing between the load-balancer nodes and the worker nodes?
- What is the relationship of the text labels about the external and internal DNSs to the worker nodes?
- What is the relationship of the external VPC network to the rest of the diagram?
- Why is there no text after the diagram to explain what it depicts, since what it depicts is not at all obvious?

Question about the caption: Is the diagram an overview of the entire CloudHub network? Or does it show only a portion of it, and that portion related somehow to two apps? The caption doesn't help me understand what the diagram depicts. There isn't any "network overview" in CloudHub, as far as I know.